addons/loyalty/models/loyalty_program.py

```diff
@@ -413,52 +413,60 @@ def get_program_templates(self):
-                'icon': 'promo_code',
-            },
-            'loyalty': {
-                'title': _('Loyalty Cards'),
-                'description': _('Win points with each purchases, and use points to get gifts'),
-                'icon': 'loyalty_cards',
-            },
-            'fidelity': {
-                'title': _('Fidelity Cards'),
-                'description': _('Buy 10 products, and get 10$ discount on the 11th one'),
-                'icon': 'fidelity_cards',
-            },
             'promotion': {
-                'title': _('Promotional Program'),
-                'description': _('Automatic promotion: 10% discount on orders higher than $50'),
+                'title': _("Promotion Program"),
+                'description': _(
+                    "Define promotions to apply automatically on your customers' orders."
+                ),
                 'icon': 'promotional_program',
             },
-            'coupons': {
-                'title': _('Coupons'),
-                'description': _('Send unique coupons that give access to rewards'),
-                'icon': 'coupons',
+            'promo_code': {
+                'title': _("Discount Code"),
+                'description': _(
+                    "Share a discount code with your customers to create a purchase incentive."
+                ),
+                'icon': 'promo_code',
             },
             'buy_x_get_y': {
-                'title': _('2+1 Free'),
-                'description': _('Buy 2 products and get a third one for free'),
+                'title': _("Buy X Get Y"),
+                'description': _(
+                    "Offer Y to your customers if they are buying X; for example, 2+1 free."
+                ),
                 'icon': '2_plus_1',
             },
             'next_order_coupons': {
-                'title': _('Next Order Coupons'),
-                'description': _('Send unique, single-use coupon code for the next purchase'),
+                'title': _("Next Order Coupons"),
+                'description': _(
+                    "Reward your customers for a purchase with a coupon to use on their next order."
+                ),
                 'icon': 'coupons',
             },
+            'loyalty': {
+                'title': _("Loyalty Cards"),
+                'description': _("Win points with each purchase, and use points to get gifts."),
+                'icon': 'loyalty_cards',
+            },
+            'coupons': {
+                'title': _("Coupons"),
+                'description': _("Generate and share unique coupons with your customers."),
+                'icon': 'coupons',
+            },
+            'fidelity': {
+                'title': _("Fidelity Cards"),
+                'description': _("Buy 10 products, and get 10$ discount on the 11th one."),
+                'icon': 'fidelity_cards',
+            },
```

Comment:
Why did you reorder the keys (and which logic did you follow) ?
Also, this method is quite shitty since it will translate a whole bunch of strings when only one is going to be used.
Could you try using the lazy translation function (not 100% sure it would work here, but it could be a nice improvement) ?

Reply:
The spec has changed lately (see chatter of the task)